tests/templates/test_layers/test_strongly_entangling.py

```diff
@@ -160,13 +160,6 @@ def circuit(weights, ranges=None):
             weights = np.random.randn(1, 2, 3)
             circuit(weights, ranges=[0])
 
-    def test_id(self):
```

Comment:
Hi @wongwsvincent how come this test was removed?

Reply:
ops, sorry! I must have mistakenly removed it when I was resolving my merge conflicts. The test is added back.